modules/apps/web-experience/export-import/export-import-service/src/main/java/com/liferay/exportimport/controller/PortletExportController.java

```diff
@@ -333,14 +333,14 @@ public void exportPortlet(
 			exportPortletPreferences(
 				portletDataContext, portletDataContext.getCompanyId(),
 				PortletKeys.PREFS_OWNER_TYPE_COMPANY, false, layout, plid,
-				portlet.getRootPortletId(), portletElement);
+				portlet.getPortletId(), portletElement);
```

Comment:
This was necessary to get the correct portlet ID for the embedded portlet. If we used "getRootPortletId", then the instance ID of the embedded portlet would be removed and the preferences would not be found.